docs/what-is-nuget.md

```diff
@@ -92,13 +92,14 @@ When you have a choice, we recommend using PackageReference. `packages.config` i
 
-- NuGet provides the central nuget.org repository with support for private hosting.
+- NuGet provides access to a public package repository, nuget.org, but also supports privately hosted respositories even on the local system.
 - NuGet provides the tools developers need for creating, publishing, and consuming packages.
 - Most importantly, NuGet maintains a reference list of packages used in a project and the ability to restore and update those packages from that list.
+- Nuget automatically manages the secondary dependencies of packages and resolves possible version conflict when a dependency is referenced multiple times by finding a compatible common version (if it can).
 
 To make these processes work efficiently, NuGet does some behind-the-scenes optimizations. Most notably, NuGet manages a package cache and a global packages folder to shortcut installation and reinstallation. The cache avoids downloading a package that's already been installed on the machine. The global packages folder allows multiple projects to share the same installed package, thereby reducing NuGet's overall footprint on the computer. The cache and global packages folder are also very helpful when you're frequently restoring a larger number of packages, as on a build server. For more details on these mechanisms, see [Managing the global packages and cache folders](consume-packages/managing-the-global-packages-and-cache-folders.md).
 
-Within an individual project, NuGet manages the overall dependency graph, which again includes resolving multiple references to different versions of the same package. It's quite common that a project takes a dependency on one or more packages that themselves have the same dependencies. Some of the most useful utility packages on nuget.org are employed by many other packages. In the entire dependency graph, then, you could easily have ten different references to different versions of the same package. To avoid bringing multiple versions of that package into the application itself, NuGet sorts out which single version can be used by all consumers. (For more information, see [Dependency Resolution](concepts/dependency-resolution.md).)
+Within an individual project, NuGet manages the overall dependency graph, which again includes resolving multiple references to different versions of the same package. It's quite common that a project takes a dependency on one or more packages that themselves have the same dependencies. Some of the most useful utility packages on nuget.org are employed by many other packages. In the entire dependency graph, then, you could easily have ten different references to different versions of the same package. To avoid bringing multiple versions of that package into the application itself, NuGet sorts out which single version can be used by all consumers. (For more information, see [Dependency Resolution](https://docs.microsoft.com/en-us/nuget/concepts/dependency-resolution).)
```

Comment:
This goes against the recommendation here: https://docs.microsoft.com/en-us/contribute/how-to-write-links#links-from-one-article-to-another. 

Please revert this.